src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work ⚙.");

```

Comment:
The message is not very informative. It should indicate the type of machine, which is 'Bulldozer' in this case, but it should also be more descriptive about the work it's doing.